features/features.go

```diff
@@ -527,7 +527,6 @@ var (
 				reportProblemsToJiraComponent("Image Registry").
 				contactPerson("flavianmissi").
 				productScope(ocpSpecific).
-				enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```

Comment:
This should not be removed, just add `configv1.Default` as an additional item in the input